src/module/helpers/tokenUtils.mjs

```diff
@@ -0,0 +1,43 @@
+/**
+ * @param {DrawSteelActor} actor
+ * @param input
+ * @param fill
+ * @returns {Promise<void>}
+ */
+export async function showFloatyText(actor, input, fill) {
+  if (!canvas.scene) {
+    return;
+  }
+
+  const [token] = actor.getActiveTokens();
+
+  if (token && (typeof input === "number")) {
+    const gridSize = canvas.scene.grid.size;
+    const scrollingTextArgs = [
+      { x: token.x + gridSize / 2, y: token.y + gridSize - 20 },
+      Math.abs(input),
+      {
+        fill: fill ?? (input < 0 ? "lightgreen" : "white"),
+        fontSize: 32,
+        stroke: 0x000000,
+        strokeThickness: 4,
+      },
+    ];
+    await token._animation;
+    await canvas.interface?.createScrollingText(...scrollingTextArgs);
```

Comment:
If we have a token, then the canvas is ready, and then `canvas.interface` is definitely ready, too.